### ISSUE TITLE
ghost restart Error on v4.25.1

### ISSUE THREAD

**Issue by Lucifer1590** [TryGhost/Ghost#13837]
### Issue Summary

After updating ghost installation with ghost update it started getting updated but in the process of "ghost restart," it throws an error and exits. After manually running "ghost restart" error still persists ,

### Steps to Reproduce

1- Update to the latest version v4.25.1 through CLI by running ghost update
2- observe error while updating

### Ghost Version

Ghost-CLI version: 1.18.1   Ghost version: 4.25.1 

### Node.js Version

v14.18.1

### How did you install Ghost?

Selfhosted server on Lionode with ubuntu 20.04LTS

### Database type

MySQL 8

### Relevant log / error output

```shell
+ sudo systemctl is-active ghost_testingstan-in
+ sudo systemctl restart ghost_testingstan-in
☴ Restarting Ghost/usr/lib/node_modules/ghost-cli/lib/utils/port-polling.js:68
                    let {message: errMsg} = message.error;
                                  ^

TypeError: Cannot destructure property 'message' of 'message.error' as it is undefined.
    at Server.<anonymous> (/usr/lib/node_modules/ghost-cli/lib/utils/port-polling.js:68:35)
    at Object.onceWrapper (events.js:519:28)
    at Server.emit (events.js:400:28)
    at emitCloseNT (net.js:1677:8)
    at processTicksAndRejections (internal/process/task_queues.js:81:21)
```

### Code of Conduct

- [X] I agree to be friendly and polite to people in this repository

**Comment by snird** [comment]
I tried printing the object of `message` by `JSON.stringify()` the result was a Info log (and not an error, thus the rejection) - but nothing in this log was useful to understand the issue.

**Comment by snird** [comment]
@Lucifer1590 temporary solution if you have your site down right now: https://github.com/TryGhost/Ghost/issues/13840

**Comment by JustroX** [comment]
+1

**Comment by Lucifer1590** [comment]
> +1

are you also facing the same issue?

**Comment by JustroX** [comment]
yes. I temporarily follow this #13840

**Comment by daivdchau2021** [comment]
+1

**Comment by yannickcof** [comment]
+1 🤒🤕

**Comment by philheijkoop** [comment]
Just want to add I am seeing this happening on new sites built through the DigitalOcean Marketplace's Ghost listing.

**Comment by LevTramana** [comment]
+1

**Comment by daniellockyer** [comment]
Hey all 👋🏻 Apologies for this. A fix has been pushed to Ghost and it will be released soon. 4.25.1 has been deprecated in NPM so it should no longer be installed.

If you've upgraded from <4.25.1, you should still have your old version in the `versions/` folder, so you can run `ghost update --rollback`.

If you freshly installed 4.25.1, or the above command doesn't work, you can run `ghost update --force` to go back to 4.25.0.

We'll be reviewing the tooling and documentation for this situation shortly 🙂

**Comment by Lucifer1590** [comment]
> ghost update --rollback

after doing this my ghost rolled back to v4.24.0 not v4.25.0 was that intentional 🤔🤔

**Comment by daniellockyer** [comment]
This is resolved in v4.26.0 so please update with Ghost-CLI 🙂
